ocaml/xapi/message_forwarding.ml

```diff
@@ -2330,7 +2330,8 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
       do_op_on ~local_fn ~__context ~host (fun session_id rpc -> Client.Host.dmesg_clear rpc session_id host)
 
     let bugreport_upload ~__context ~host ~url ~options =
```

Comment:
I think `do_upload` in Xapi_support neeads the same treatment wrt to URL and proxy.